cmd/podman/utils/alias.go

```diff
@@ -23,8 +23,6 @@ func AliasFlags(f *pflag.FlagSet, name string) pflag.NormalizedName {
 		name = "ns"
 	case "storage":
 		name = "external"
-	case "purge":
-		name = "rm"
```

Comment:
Why is this removed? It will effect all commands that use this alias function, i.e. `podman run --purge` would no longer work

Reply:
This was added purely for the alias of this command, 
docker run --purge does not exists, so no equivalent to podman run --purge.

If I leave the alias, then the -p shows up in `podman manifest push --help`
I added the alias to specifically fix this problem, so removing it and getting the -p to work as an alias, works better.

Reply:
Sure I understand why the flag is added. Just saying removing it from the alias will affect other commands and this could break users. I know it is not documented but I see little gain by removing it vs the risk of breaking changes.

Reply:
The gain is that cobra is not handling it properly, and is documenting the `-p, rm ...`
Which then requires us to document the -p in the man page, because hack/xref-helpmsgs-manpages complains about it not being documented.
The fact that `podman run --purge` worked, could be considered a bug, I see no reason to make the tool worse just because of the infinitesimally small chance, that someone used an undocumented option. Where said option is not only undocumented for Podman but also for Docker.

Reply:
mhh, yeah that sounds like a bug in cobra/pflag
If this is required to make it work I am fine with it.